DigitalLearningSolutions.Web/ViewModels/TrackingSystem/Delegates/DelegateProgress/EditDelegateCourseAdminFieldViewModel.cs

```diff
@@ -0,0 +1,75 @@
+﻿namespace DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates.DelegateProgress
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using DigitalLearningSolutions.Data.Models.Courses;
+    using DigitalLearningSolutions.Web.Models.Enums;
+    using DigitalLearningSolutions.Web.ViewModels.Common.ViewComponents;
+
+    public class EditDelegateCourseAdminFieldViewModel : EditDelegateCourseAdminFieldFormData
+    {
+        public EditDelegateCourseAdminFieldViewModel(
+            int progressId,
+            int promptNumber,
+            DelegateCourseDetails details,
+            DelegateAccessRoute accessedVia,
+            int? returnPage
+        ) : base(details, returnPage)
+        {
+            var courseAdminField = details.CourseAdminFields.Find(c => c.PromptNumber == promptNumber);
+
+            Options = courseAdminField!.Options;
+            Answer = GetAnswer(details.DelegateCourseInfo, promptNumber);
+            Radios = GetRadioItems(details.DelegateCourseInfo, promptNumber);
+            PromptText = courseAdminField.PromptText;
+            ProgressId = progressId;
+            AccessedVia = accessedVia;
+            ReturnPage = returnPage;
+        }
+
+        public EditDelegateCourseAdminFieldViewModel(
+            EditDelegateCourseAdminFieldFormData formData,
+            DelegateCourseDetails details,
+            int progressId,
+            int promptNumber,
+            DelegateAccessRoute accessedVia,
+            int? returnPage

```

Comment:
unused param

Reply:
passed to base formData

Reply:
Yeah that sounds right. Removed it as a param